### PR TITLE
Remove incompatible options from `systemd-run`

### LINKED ISSUE #627: Docker credential helper for AWS ECR not being installed
### CI/CD Environment Variables
| Name | Value |
| -- | -- |
| `DOCKER_AUTH_CONFIG` | `{"credsStore": "ecr-login"}` |
| `AWS_ACCESS_KEY_ID` | [AWS access key identifier](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html) |
| `AWS_SECRET_ACCESS_KEY` | [AWS secret access key](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html) |
| `AWS_SESSION_TOKEN` | [AWS session token (optional)](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html) |
| `REPO_TOKEN` | [GitLab Personal Access Token](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html) | 

### `.gitlab-ci.yml`
```yaml
runner:
  when: always
  image: iterativeai/cml
  script:
  - >-
    cml runner
    --cloud=aws
    --cloud-startup-script=$(echo 'curl https://amazon-ecr-credential-helper-releases.s3.us-east-2.amazonaws.com/0.5.0/linux-amd64/docker-credential-ecr-login --output /usr/bin/docker-credential-ecr-login && chmod a+x /usr/bin/docker-credential-ecr-login' | base64 --wrap=0)
    --labels=cml
job:
  tags: [cml]
  needs: [runner]
  image: 000000000000.dkr.ecr.us-east-1.amazonaws.com/example:latest
  script:
  - echo hello
```

## Issue

The lengthy `--cloud-startup-script` option in the example above should not be needed.

https://github.com/iterative/terraform-provider-iterative/blob/28ce78188618771e3338ef4373b295c6a8e85f2b/environment/setup.sh#L21-L23

### PR DESCRIPTION
**Pull request by 0x2b3bfa0**
Closes #627

### CODE REVIEW

**Review by DavidGOrtega — APPROVED** [review]
👍 

disclaimer: Not tested on my side 🙈

**Review by dacbd — APPROVED** [review]

File: environment/setup.sh [hunk 1]
```diff
@@ -20,7 +20,7 @@ if [ ! -f "$FILE" ]; then
 
   get_ecr_helper="curl https://amazon-ecr-credential-helper-releases.s3.us-east-2.amazonaws.com/0.5.0/linux-amd64/docker-credential-ecr-login --output /usr/bin/docker-credential-ecr-login"
   chmod_ecr_helper="chmod 755 /usr/bin/docker-credential-ecr-login"
-  sudo systemd-run --same-dir --no-block --service-type=exec bash -c "$get_ecr_helper && $chmod_ecr_helper"
+  sudo systemd-run --no-block bash -c "$get_ecr_helper && $chmod_ecr_helper"
```
Comment (dacbd) [inline environment/setup.sh:23]: `%s/x/y/g` issues?

**Review by dacbd — CHANGES_REQUESTED** [review]

File: environment/setup.sh (hunk 1 above)
Comment (dacbd) [reply environment/setup.sh:23]: perhaps from a rebase mistake?

Comment (0x2b3bfa0) [reply environment/setup.sh]: PEBCAK

**Review by dacbd — APPROVED** [review]

**Pull request merged**